web/islands/TestWebhookDialog.tsx

```diff
@@ -41,51 +51,33 @@ export const TestWebhookDialog = ({ setToast, ...props }: TestWebhookProps) => {
         <form onSubmit={handleSubmit} class="form-control w-full">
           <label class="label">
             Transaction Hash
-            <input
-              type="text"
-              name="txHash"
-              class="input input-bordered w-full max-w-xs"
-            />
+            <Input name="txHash" />
           </label>
           <label class="label">
             Block Hash
-            <input
-              type="text"
-              name="blockHash"
-              class="input input-bordered w-full max-w-xs"
-            />
+            <Input name="blockHash" />
           </label>
           <label class="label">
             Topic 1
-            <input
-              type="text"
-              name="topic1"
-              class="input input-bordered w-full max-w-xs"
-            />
+            <Input name="topic1" />
           </label>
           <label class="label">
             Topic 2
-            <input
-              type="text"
-              name="topic2"
-              class="input input-bordered w-full max-w-xs"
-            />
+            <Input name="topic2" />
           </label>
           <label class="label">
             Topic 3
-            <input
-              type="text"
-              name="topic3"
-              class="input input-bordered w-full max-w-xs"
-            />
+            <Input name="topic3" />
           </label>
           <label class="label">
             Data
-            <input
-              type="text"
-              name="data"
-              class="input input-bordered w-full max-w-xs"
-            />
+            <Input name="data" />
```

Comment:
Actually, data can be any size of data.

Reply:
I'll change the validation more flexible for data, it seems viem requires data size to be bigger or equal than 32 bytes however